docs/guide/examples.rst

```diff
@@ -158,6 +158,22 @@ Multiprocessing: Unleashing the Power of Vectorized Environments
           env.render()
 
 
+Multiprocessing with off-policy algorithms
```

Comment:
A note about the potential "worse sample efficiency" etc would be nice here. I expect people try increasing number of envs but observe no big speedups in training (depending on the env). A link to the PR commente/experiment?